### DISCUSSION TITLE
STIG Manager Accreditation

### DISCUSSION THREAD

**Discussion by Vesfoldy** [NUWCDIVNPT/stig-manager#1160] [General]
Is the STIG Manager software accredited on a DoD network?  If so, what is the eMass package name and number?

**Comment by csmig** [comment]
STIG Manager deployments have been accredited by several DoD organizations. You'll need to contact whichever organization hosts your deployment for their specific eMass information. Software projects are not accredited or granted ATO, but most organizations require the software be approved for use before it can be deployed. For example, if you are subject to the Navy's DADMS requirements you would want to know that STIG Manager's DADMS ID is 131584.

If you wish to deploy an instance of STIG Manager on a DoD network , then we recommend familiarity with the chapter [Securing and Assessing STIG Manager Deployments](https://stig-manager.readthedocs.io/en/latest/installation-and-setup/securing.html) in our documentation.

**Reply by Vesfoldy** [reply]
Thanks csmig.  I was hoping someone could share an eMass ID I could use for reciprocity for our accreditation effort.  Anyway, I will see what I can glean from looking up the DADMS ID.  Thanks.

**Reply by TWALKER273** [reply]
Vesfoldy! Were you able to gather any information in your efforts to get an ATO?

**Comment by cd-rite** [comment]
@Vesfoldy If you are on Navy Flank Speed, you can join the STIGMan OSS Team with this code: qrym6dn
If you ask your question in the "Help" channel there someone else with a deployment may be able to point you at their eMASS info.

**Reply by Vesfoldy** [reply]
@cd-rite I am on Navy Flank Speed.  I will join and post my question in the "Help" channel. Thanks.
